source/community/reactnative/src/models/container-model.js

```diff
@@ -0,0 +1,155 @@
+            this.backgroundImage = payload.backgroundImage;;
+        }
+        this.verticalContentAlignment = payload.verticalContentAlignment;
+        this.style = payload.style;
+        this.bleed = payload.bleed;
+    }
+}
+
+export class AdaptiveCardModel extends BaseContainerModel {
+    constructor(parent, payload) {
+        super(parent, payload);
+        this.type = ElementType.AdaptiveCard;
+        this.fallbackText = payload.fallbackText;
+        this.version = payload.version;
+        this.speak = payload.speak;
+        this.children = [];
+        this.actions = [];
+        this.children.push(...ModelFactory.createGroup(this, payload.body));
+        this.actions.push(...ModelFactory.createGroup(this, payload.actions));
+        this.show = true;
+    }
+}
+
+export class ContainerModel extends BaseContainerModel {
+    constructor(parent, payload) {
+        super(parent, payload);
+        this.type = ElementType.Container;
+        this.children = [];
+        this.children.push(...ModelFactory.createGroup(this, payload.items));
+        this.height = payload.height;
+    }
+
+    get items(){
+        return this.children;
+    }
+}
+
+export class ColumnSetModel extends BaseContainerModel {
+    constructor(parent, payload) {
+        super(parent, payload);
+        this.type = ElementType.ColumnSet;
+        this.children = [];
+        if (payload.columns) {
+            payload.columns.forEach((item) => {
+                let column = new ColumnModel(this, item);
+                if (column) {
+                    this.children.push(column);
+                }
+            });
+        }
+        this.height = payload.height;
+    }
+    get columns() {
+        return this.children;
+    }
+}
+
+export class ColumnModel extends BaseContainerModel {
+    constructor(parent, payload) {
+        super(parent, payload);
+        this.type = ElementType.Column;
+        this.children = [];
+        this.children.push(...ModelFactory.createGroup(this, payload.items));
+        this.height = payload.height;
+        if (payload.width) {
+            if (payload.width === 'auto' || payload.width === 'stretch') {
+                this.width = payload.width;
+            }
+            else {
+                let columnWidth = parseInt(payload.width, 10);
+                if (columnWidth < 0) {
+                    columnWidth = 0;
+                }
+                this.width = columnWidth;
+            }
+        }
+    }
+    get items() {
+        return this.children;
+    }
+}
+
+export class FactModel {
+    constructor(payload) {
+        this.type = 'Fact';
+        this.title = payload.title;
+        this.value = payload.value;
+    }
+}
+
+export class FactSetModel extends BaseContainerModel {
+    constructor(parent, payload) {
+        super(parent, payload);
+        this.type = ElementType.FactSet;
+        this.children = [];
+        if (payload.facts) {
+            payload.facts.forEach((item) => {
```

Comment:
Can't we user ModelFactory.createGroup here too?

Reply:
Same reason as above.